### PR TITLE
Feature/tmbadge

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
Screenshot_1691573784
Screenshot_1691573787
Screenshot_1691573789

### CODE REVIEW

File: Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Name="tmBadge"
+             x:Class="Trimble.Modus.Components.TMBadge">
+  <Border
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml:6]: Remove the empty line here

File: DemoApp/DemoApp/Views/TMBadgesPage.xaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <Label Text="Rounded Badges"/>
+    <StackLayout Orientation="Horizontal" Spacing="10" >
+      <modus:TMBadge Text="Text" Color="Primary" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Secondary" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Tertiary" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Success" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Warning" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Error" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+    </StackLayout>
+    <modus:TMRadioButtonGroup Orientation="Horizontal" SelectedRadioButtonChangedCommand="{Binding BadgeSizeChangedCommand}">
```
Comment (rthanga1) [inline DemoApp/DemoApp/Views/TMBadgesPage.xaml:26]: This radio button group doesn't have any effect when i ran

**Review by rthanga1 — CHANGES_REQUESTED** [review]

File: Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+                tmBadge.frame.StrokeShape = new RoundRectangle
+                {
+                    CornerRadius = new CornerRadius(4)
+                };
+                break;
+        }
+    }
+
+    private static void UpdateLabelOnSize(TMBadge badge)
+    {
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml.cs:103]: The Size and Color can be set in xaml via binding the properties but anyway those properties must be set in the code-behinid.

File: Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml (hunk 1 above)
Comment (ElishaSamPeterPrabhu) [reply Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml]: There is no empty line from line 5-7?

File: Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Name="tmBadge"
+             x:Class="Trimble.Modus.Components.TMBadge">
+  <Border 
+          x:Name="frame"
+          HorizontalOptions="Start"
+          VerticalOptions="Center"
+          Stroke="Transparent"
+          >
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml]: removed

File: DemoApp/DemoApp/Views/TMBadgesPage.xaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      <modus:TMBadge Text="Text" Color="Primary" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Secondary" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Tertiary" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Success" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Warning" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Error" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+    </StackLayout>
+    <Label Text="Rounded Badges"/>
+    <StackLayout Orientation="Horizontal" Spacing="10" >
+      <modus:TMBadge Text="Text" Color="Primary" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Views/TMBadgesPage.xaml:19]: Changed

File: DemoApp/DemoApp/Views/TMBadgesPage.xaml (hunk 2 above)
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Views/TMBadgesPage.xaml]: The initial ChipSize is set to Large , But the initial radio button is set as medium . Now set both to medium and can see a visible change after the first change

**Review by rthanga1 — APPROVED** [review]

File: Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml (hunk 1 above)
Comment (rthanga1) [reply Trimble.Modus.Components/Controls/TMBadge/TMBadge.xaml]: I fixed it.

**Review by rthanga1 — APPROVED** [review]

**Review by BAL4-R — APPROVED** [review]

**Pull request merged**